PatientWebApplication/SearchMicroserviceApi/Model/Room.cs

```diff
@@ -0,0 +1,44 @@
+/***********************************************************************
+ * Module:  Room.cs
+ * Author:  Lenovo
+ * Purpose: Definition of the Class Manager.Room
+ ***********************************************************************/
+
+using System.Collections.Generic;
+
+namespace SearchMicroserviceApi.Model
+{
+    public class Room : Entity
+    {
+        public string typeOfRoom { get; set; }
+
+        public virtual List<ModelEquipment> equipment { get; set; }
+        public virtual List<ModelMedicine> medicine { get; set; }
+        public bool forUse { get; set; }
```

Comment:
Write class properties to start with upper letter and remove "this" in constructors